src/subclass/application.rs

```diff
@@ -0,0 +1,530 @@
+// Copyright 2019, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use gio_sys;
+use glib_sys;
+
+use glib::translate::*;
+
+use glib::subclass::prelude::*;
+
+use Application;
+use ApplicationClass;
+
+use libc::{c_char, c_void};
+use std::convert;
+use std::ffi::OsString;
+use std::fmt;
+use std::ops::Deref;
+use std::ptr;
+
+pub struct ArgumentList {
```

Comment:
Why not move `ArgumentList` to separate file?

Reply:
Because it will only ever be used inside this file